### PR TITLE
switch to hex integers for eth_feeHistory

### PR DESCRIPTION
**Pull request by RatanRSur**
https://github.com/ethereum/eth1.0-specs/blob/master/json-rpc/spec.json#L200

### CODE REVIEW

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/FeeHistory.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    @Nullable
+    @JsonProperty("reward")
+    List<List<String>> getReward();
+
+    static FeeHistoryResult from(final FeeHistory feeHistory) {
+      return ImmutableFeeHistoryResult.of(
+          "0x" + Long.toHexString(feeHistory.getOldestBlock()),
+          feeHistory.getBaseFeePerGas().stream()
+              .map(baseFeePerGas -> "0x" + Long.toHexString(baseFeePerGas))
```
Comment (matkt) [inline ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/FeeHistory.java:59]: Why not using Quantity.create to have the hex string value in this case ?

**Review by matkt — CHANGES_REQUESTED** [review]
Small comment

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/FeeHistory.java (hunk 1 above)
Comment (RatanRSur) [reply ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/FeeHistory.java:59]: Oh right! Forgot that existed. Will do.

**Review by matkt — APPROVED** [review]
LGTM

**Pull request merged**